usr/share/rear/lib/framework-functions.sh

```diff
@@ -140,15 +140,11 @@ function cleanup_build_area_and_end_program () {
         # in worst case it could not umount; so before remove the BUILD_DIR check if above outputfs is gone
         if mountpoint -q "$BUILD_DIR/outputfs" ; then
             # still mounted it seems
-            LogPrint "Directory $BUILD_DIR/outputfs still mounted - trying lazy umount"
             sleep 2
-            umount -f -l $BUILD_DIR/outputfs >&2
-            rm -Rf $v $BUILD_DIR/outputfs >&2
-        else
-            # not mounted so we can safely delete $BUILD_DIR/outputfs
-            rm -Rf $BUILD_DIR/outputfs
+            umount_mountpoint_lazy $BUILD_DIR/outputfs
         fi
-        rm -Rf $v $BUILD_DIR >&2
+        remove_temporary_mountpoint '$BUILD_DIR/outputfs' || BugError "Directory $BUILD_DIR/outputfs not empty, can not remove"
```

Comment:
I think the Englisch spelling is always `cannot` (and never `can not`)
but I am not a native English speaker.

Reply:
https://en.wiktionary.org/wiki/cannot#Usage_notes , I can change it though

Reply:
Again thanks for the explanation!
And again I learned something new.
Of course you can keep it as is.